bsp/esp_bsp_generic/Kconfig

```diff
@@ -479,7 +518,285 @@ menu "Board Support Package"
+        
+        menu "Rotation"
+            depends on BSP_DISPLAY_ENABLED
+
+            config BSP_DISPLAY_ROTATION_SWAP_XY
+                bool "Swap XY"
+                default n
+                help
+                    Swap X and Y axis on the screen.
+
+            config BSP_DISPLAY_ROTATION_MIRROR_X
+                bool "Mirror X"
+                default n
+                help
+                    Mirror X axis on the screen.
+
+            config BSP_DISPLAY_ROTATION_MIRROR_Y
+                bool "Mirror Y"
+                default n
+                help
+                    Mirror Y axis on the screen.
+        endmenu
+            
+        choice  
+            prompt "Color space"
+            depends on BSP_DISPLAY_ENABLED
+            help
+                Select a display color space 
+            config BSP_DISPLAY_COLOR_SPACE_RGB
+                bool "RGB"
+            config BSP_DISPLAY_COLOR_SPACE_BGR
+                bool "BGR"
+        endchoice
+
+        config BSP_DISPLAY_CMD_BITS
+            depends on BSP_DISPLAY_ENABLED
+            int "LCD command bits"
+            default 8
+            help
+                The command bits of the display.
+
+        config BSP_DISPLAY_PARAM_BITS
+            depends on BSP_DISPLAY_ENABLED
+            int "LCD param bits"
+            default 8 if BSP_DISPLAY_DRIVER_ILI9341
+            default 8 if BSP_DISPLAY_DRIVER_ST7789
+            default 8 if BSP_DISPLAY_DRIVER_GC9A01
+            help
+                The param bits of the display.
+            
+        config BSP_DISPLAY_PIXEL_CLOCK
+            depends on BSP_DISPLAY_ENABLED
+            int 
+            prompt "Display pixel clock (MHz)"
+            default 20
+            range 1 40
+            help
+                The pixel clock of the display in MHz.
+            
+        config BSP_DISPLAY_WIDTH
+            depends on BSP_DISPLAY_ENABLED
+            int 
+            prompt "Display width"
+            default 320
+            help
+                The width in pixels of the display.
+            
+        config BSP_DISPLAY_HEIGHT
+            depends on BSP_DISPLAY_ENABLED
+            int 
+            prompt "Display height"
+            default 240
+            help
+                The height in pixels of the display.
+                
+        config BSP_DISPLAY_BRIGHTNESS_LEDC_CH
+            depends on BSP_DISPLAY_ENABLED && BSP_DISPLAY_BACKLIGHT_GPIO != -1
+            int "LEDC channel index for brightness control"
+            default 1
+            range 0 7
+            help
+                LEDC channel is used to generate PWM signal that controls display brightness.
+                Set LEDC index that should be used.
+
+        config BSP_LCD_DRAW_BUF_HEIGHT
+            depends on BSP_DISPLAY_ENABLED
+            int "LCD framebuf height"
+            default 100
+            range 10 240
+            help
+                Framebuf is used for lvgl rendering output.
```

Comment:
Could you please add information about units of this value? eg. row, bytes...

Reply:
added